script.js

```diff
@@ -64,59 +64,91 @@ function resetGame() {
     pumpkinMove.pause();
     reaperMove.cancel();
     grimReaperZone.removeEventListener('mouseleave', moveReaper);
+
 }
 
 // code here to activate win modal displaying
 function win() {
     winModal.style.display = "block";
+    clearInterval(timer);
+    timerRunning = false;
     resetGame();
 }
 
 // code here to activate lose modal displaying
 function lose() {
     loseModal.style.display = "block";
+    clearInterval(timer);
+    timerRunning = false;
     resetGame();
 }
 
 winModal.addEventListener('click', (e) => {
     e.stopPropagation();
     winModal.style.display = "none";
+    document.querySelector('#timer p').textContent = '00:45';
```

Comment:
Using `'#timer p'` as query is a little bit dependent on HTML - should probably just put an id on the exact element you need

Reply:
This is in multiple places